### PR TITLE
Kubernetes configuration for initial import to postgres

### PR DESCRIPTION
**Pull request by amatissart**

### CODE REVIEW

File: kubernetes/README.MD [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
- 
- these two containers shares pbfs volume located (/tmp/pbfs)
- 
- postgres.rc
-  - replicate {one} postgres pod (should be a static set or whatever)
+
+configmap.yml
+ - a default configuration for jobs and services
+
+load_db.jod.yml
```
Comment (jbgriesner) [inline kubernetes/README.MD]: "job" :)

File: kubernetes/configmap.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: default-config
+  namespace: default
+data:
+  pbf_url: "https://download.geofabrik.de/europe/luxembourg-latest.osm.pbf"
+  redis_set_key: "" # If not empty, this key will be set to 'true' after import to postgres
```
Comment (antoine-de) [inline kubernetes/configmap.yml]: why not a boolean value here ?

File: kubernetes/load_db.job.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: initial-import
+spec:
+  template:
+    spec:
+      nodeSelector:
+        maps-pg-importer: "true"
```
Comment (antoine-de) [inline kubernetes/load_db.job.yml]: I think we should add a comment here since it's quite important

File: load_db/Dockerfile [hunk 4]
```diff
@@ -2,7 +2,8 @@ FROM python:3.6-stretch
 
 ENV MAIN_DIR=/srv/import_data \
     SQL_DIR=/srv/import_data/sql \
-    DATA_DIR=/tmp/pbfs
+    DATA_DIR=/tmp/pbfs \
```
Comment (antoine-de) [inline load_db/Dockerfile]: I prefer /data as the path

File: load_db/import_data.sh [hunk 5]
```diff
@@ -17,11 +17,9 @@ psql -Xq -h $host -U $user -d $database --set ON_ERROR_STOP="1" -c "CREATE TABLE
...
-
-
 INVOKE_DATA_DIR=$DATA_DIR INVOKE_OSM_FILE=$osm_file pipenv run invoke
 
 # we tell redis that the import is finished so tilerator can start
-if [ -n "$REDIS_SET_KEY" ]; then
-	redis-cli -h redis set "$REDIS_SET_KEY" 'true'
+if [ "$REDIS_SET_KEY" = "true" ]; then
+	redis-cli -h redis set 'data_imported' 'true'
 fi
```
Comment (antoine-de) [inline load_db/import_data.sh]: missing newline

**Pull request merged**